src/control/conf.h

```diff
@@ -215,7 +215,7 @@ static inline void dt_conf_init(dt_conf_t *cf, const char *filename, GSList *ove
     if(!i)
     {
       snprintf(darktable.conf->filename, sizeof(darktable.conf->filename), "%s", filename);
-      f = fopen(filename, "rb");
+      f = fopen(filename, "r");
```

Comment:
I am fine with that. Actually, I did that on purpose to make it easy/feasible to share the config file between computers running different operating systems.